modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -121,6 +121,8 @@
 				[@subpath, @path, foo]
 				[, res://test.gd, bar]
 				[/codeblock]
+				[b]Note:[/b] This function is not recursive, which means that nested objects will not be represented as dictionaries. Also, properties of reference types ([Object], [Dictionary], [Array], and packed arrays) are copied by reference, not duplicated.
+				[b]Note:[/b] This function does not support non-GDScript instances and objects without an attached script.
```

Comment:
Nasty double-negatives. hmm... assuming I understand right:
```suggestion
				[b]Note:[/b] This function only supports objects with a [GDScript] attached.
```
or
```suggestion
				[b]Note:[/b] This function only supports objects with a script written in GDScript attached.
```
There's also this pesky note above which I feel like could be removed in favour of mentioning this information below
> [b]Note:[/b] Cannot be used to serialize objects with built-in scripts attached or objects allocated within built-in scripts.